code/game/objects/items/plushes.dm

```diff
@@ -504,115 +552,12 @@
 	attack_verb = list("clawed", "hissed", "tail slapped")
 	squeak_override = list('sound/weapons/slash.ogg' = 1)
 
-/obj/item/toy/plush/lizardplushie/durgit
-	icon_state = "durgit"
-	item_state = "durgit"
-	squeak_override = list('modular_citadel/sound/voice/weh.ogg' = 1) //Durgit's the origin of the sound
-
-/obj/item/toy/plush/lizardplushie/rio
-	icon_state = "rio"
-	item_state = "rio"
-
-/obj/item/toy/plush/lizardplushie/dan
-	icon_state = "dan"
-	item_state = "dan"
-
-/obj/item/toy/plush/lizardplushie/urinsu
-	icon_state = "urinsu"
-	item_state = "urinsu"
-
-/obj/item/toy/plush/lizardplushie/arfrehn
-	icon_state = "arfrehn"
-	item_state = "arfrehn"
-
-/obj/item/toy/plush/lizardplushie/soars
-	icon_state = "soars"
-	item_state = "soars"
-
-/obj/item/toy/plush/lizardplushie/ghostie
-	icon_state = "ghostie"
-	item_state = "ghostie"
-
-/obj/item/toy/plush/lizardplushie/amber
-	icon_state = "amber"
-	item_state = "amber"
-
-/obj/item/toy/plush/lizardplushie/cyan
-	icon_state = "cyan"
-	item_state = "cyan"
-
-/obj/item/toy/plush/lizardplushie/meena
-	icon_state = "meena"
-	item_state = "meena"
-
-/obj/item/toy/plush/lizardplushie/stalks
-	icon_state = "stalks"
-	item_state = "stalks"
-
 /obj/item/toy/plush/lizardplushie/kobold
```

Comment:
Might be another player reference, or maybe just someone's undying love for cutebolds.